leo/commands/package/logout.rs

```diff
@@ -14,40 +14,38 @@
 // You should have received a copy of the GNU General Public License
 // along with the Leo library. If not, see <https://www.gnu.org/licenses/>.
 
-//
-// Usage:
-//
-//  leo logout
-//
+use crate::{commands::Command, context::Context};
 
-#[derive(Debug)]
-pub struct LogoutCommand;
-
-use crate::{cli::CLI, cli_types::*, config::remove_token, errors::CLIError};
+use crate::config::remove_token;
+use anyhow::Result;
```

Comment:
nit: please organize the imports here into two sections - leo specific crates & external crates